core/common/src/main/java/alluxio/Constants.java

```diff
@@ -111,9 +110,8 @@
   public static final String FILE_SYSTEM_WORKER_CLIENT_SERVICE_NAME = "FileSystemWorkerClient";
   public static final String KEY_VALUE_MASTER_CLIENT_SERVICE_NAME = "KeyValueMasterClient";
   public static final String KEY_VALUE_WORKER_CLIENT_SERVICE_NAME = "KeyValueWorkerClient";
-  public static final String FILE_SYSTEM_CLIENT_SERVICE_NAME = "FileSystemClient";
 
-  public static final String REST_API_PREFIX = "/v1/api";
+  public static final String REST_API_PREFIX = "/api/v1";
```

Comment:
I think it is fine if clients of the internal v1.3 REST API are not compatible with v1.4 REST API

but newer clients should be still able to talk to v1.3 servers by checking for the version first and the figuring out which API to us